suse_migration_services/units/migrate.py

```diff
@@ -40,7 +40,13 @@ def main():
     Logger.setup()
     log = logging.getLogger(Defaults.get_migration_log_name())
     root_path = Defaults.get_system_root_path()
-
+    zypper_log = os.sep.join(
+        [root_path, 'var', 'log', 'zypper.log']
+    )
+    log.info('Bind mounting /var/log/zypper.log')
+    Command.run(
+        ['mount', '--bind', zypper_log, '/var/log/zypper.log']
+    )
```

Comment:
The bind mount is nice but it has to check if the destination '/var/log/zypper.log' exists and if not create it. It also should not mount if zypper_log does not exist

Apart from that this is the wrong place for this mounting code. It belongs to units/prepare.py

Reply:
I also forgot to mention that I think the mount should not cause the system to fail. Thus I would use raise_on_error=False in the Command call

Reply:
You could spare the check if zypper_log exists if you use raise_on_error=False, except if you want to add some descriptive error messages on the condition. I leave that up to you

Reply:
I think I would just log the raise message and be done

Reply:
> The bind mount is nice but it has to check if the destination '/var/log/zypper.log' exists and if not create it. It also should not mount if zypper_log does not exist
> 
> Apart from that this is the wrong place for this mounting code. It belongs to units/prepare.py

I thought to put it in prepare but that code deals with the information about SCC, RMT or SMT.
So I thought it wouldn't be its responsibility. I will update it.